docs/source/guide/scim_setup.md

```diff
@@ -115,7 +115,7 @@ The most convenient way to manage access to the application is via groups. You c
         Select an existing workspace or create a new one. You can map multiple groups to the same workspace.  
     * **Projects to Groups Mapping**---Map groups to roles at the project level. Project-level roles can be **Annotator**, **Reviewer**, or **Inherit**. You can map multiple groups to the same roles and the same projects. For more information on roles, see [Roles in Label Studio Enterprise](manage_users#Roles-in-Label-Studio-Enterprise). 
     
-        If you select **Inherit**, the group will inherit the role set above under **Organization Roles to Groups Mapping.** If the group is inheriting the **Not Activated** role, they are still added to the project, but they cannot be assigned tasks. 
+        If you select **Inherit**, the group will inherit the role set above under **Organization Roles to Groups Mapping.** If the group is inheriting the Not Activated role, the users are mapped to the project, but they are not actually assigned to the project until the group is synced (meaning that the user authenticates with SSO). 
```

Comment:
> meaning that the user authenticates with SSO

This actually will run on SCIM group push from the IdP not on SSO auth of the user as SCIM is a provisioner of accounts.

Reply:
Ok, removed the SSO bit and just made it generic.